config/samples/dataplane_v1beta1_openstackdataplanerole.yaml

```diff
@@ -3,16 +3,6 @@ kind: OpenStackDataPlaneRole
 metadata:
   name: openstackdataplanerole-sample
 spec:
-  dataPlaneNodes:
-    - name: openstackdataplanenode-sample
-      hostName: openstackdataplanenode-sample.localdomain
-      ansibleHost: 192.168.122.18
-      node:
-        networks:
-          - network: ctlplane
-            fixedIP: 192.168.122.18
-    - name: openstackdataplanenode-sample-from
-      nodeFrom: openstackdataplanenode-sample-from
```

Comment:
I like the idea of declaring multiple node in one shot like this, but I understand the complexity of dealing with multiple reconcile loops